packages/p2p-v2/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { useDevice } from '../../hooks';
+import './Input.scss';
+
+type TInputProps = {
+    errorMessage?: string;
+    hasError?: boolean;
+    onBlur?: () => void;
+    onChange?: () => void;
+    placeholder?: string;
+    ref?: React.Ref<HTMLInputElement> | React.RefCallback<HTMLInputElement>;
+    value?: string;
+};
+
+const Input = ({ errorMessage, hasError, onBlur, onChange, placeholder, ref, value }: TInputProps) => {
```

Comment:
@ameerul-deriv, Wouldn't it be better to use a `forwardRef` here instead of passing the `ref` as part of the `props`?